File: Exercicios-Beecrowd/Kotlin/1035.kt
import java.util.*

fun main(args: Array<String>) {
    val list= readLine().toString().split(" ")
    if (list != null) {
        val a = list[0].toInt()
        val b = list[1].toInt()
        val c = list[2].toInt()
        val d = list[3].toInt()





        if ((b > c) && (d > a) && (c + d > a + b) && (c > 0) && (d > 0) && (a % 2 == 0)) {
            println("Valores aceitos")

        } else {
            println("Valores nao aceitos")
        }

    }
    else{
        println("erro")
    }
}

File: Exercicios-Beecrowd/C#/1035.cs
using System; 

class URI {

    static void Main(string[] args) { 

        string z = Console.ReadLine().Replace(" ","");
        int a = Convert.ToInt32(z[0]);
        int b = Convert.ToInt32(z[1]);
        int c = Convert.ToInt32(z[2]);
        int d = Convert.ToInt32(z[3]);
        if (b>c && d>a && c+d>a+b && c>0  && d>0 && a%2==0) {
            Console.WriteLine("Valores aceitos");
        }
        else
        {
            Console.WriteLine("Valores nao aceitos");
        }

    }

}